__tests__/app.test.js

```diff
@@ -377,18 +379,28 @@ describe("GET /api/articles?topic=", () => {
       .expect(200)
       .then((res) => {
         const { articles } = res.body;
+
         expect(articles.length > 0).toBe(true);
 
         articles.forEach((article) => {
           expect(article).toHaveProperty("topic", expect.any(String));
           expect(article).toHaveProperty("article_id", expect.any(Number));
           expect(article).toHaveProperty("title", expect.any(String));
           expect(article).toHaveProperty("author", expect.any(String));
-          expect(article).toHaveProperty("body", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(String));
           expect(article).toHaveProperty("created_at", expect.any(String));
           expect(article).toHaveProperty("votes", expect.any(Number));
           expect(article).toHaveProperty("article_img_url", expect.any(String));
         });
       });
   });
+  test("400: returns appropriate response when sending a get request with a topic that contains no articles", () => {
+    return supertest(app)
+      .get("/api/articles?topic=paper")
```

Comment:
I would suggest that this should be a 200 and responds with an empty array! Think about it - an article _could_ be added right?